src/lib/theme/plugin.ts

```diff
@@ -76,6 +76,9 @@ const temporal = plugin(
           backgroundColor: css('--color-interactive-ghost-active'),
         },
       },
+      '.surface-information': {
+        backgroundColor: css('--color-surface-information'),
```

Comment:
Just wanted to follow-up here. Was there a reason `surface-information` was added here? Looks like we're just using `backgroundColor` and that already exists ⬇️ 

https://github.com/temporalio/ui/blob/5f41480f10ace77f9ee0e31102402786583ca606/src/lib/theme/plugin.ts#L131

Reply:
I've gone ahead and removed it [here](https://github.com/temporalio/ui/pull/2248/commits/354df4f9b8f002c78797c53e83e9301929a4a558).